### PR TITLE
*: reduce memory consumption by reducing channel sizes

### PR DESCRIPTION
**Pull request by liuzix**
### What problem does this PR solve? 
- TiCDC can OOM frequently on machines with memory less than 16GB.

### What is changed and how it works?
- Decreased the channel sizes in two places.

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test 

Side effects

 - Possible performance regression

Related changes

 - Need to cherry-pick to the release branch

### Release note

- Reduce unnecessary memory consumption

### CODE REVIEW

**Review by amyangfei — APPROVED** [review]

File: cdc/kv/region_worker.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ var (
 	// other components in TiCDC, including worker pool task chan size, mounter
 	// chan size etc.
 	// TODO: unified channel buffer mechanism
-	regionWorkerInputChanSize = 12800
+	regionWorkerInputChanSize = 128
```
Comment (amyangfei) [inline cdc/kv/region_worker.go:48]: The smaller `regionWorkerInputChanSize` will tend to use region worker pool

**Review by ben1009 — APPROVED** [review]

**Pull request merged**